playground/mocks/data/idp/idx/error-authenticator-enroll-phone-invalid-number.json

```diff
@@ -1,6 +1,6 @@
 {
   "version":"1.0.0",
-  "stateHandle":"02A2CNDXsH6luRQV4R-GOWt2_ynSdb3UWg-3Ce_-sN",
+  "stateHandle":"01OCl7uyAUC4CUqHsObI9bvFiq01cRFgbnpJQ1bz82",
```

Comment:
Had to change stateHandle due to auth-js expecting the saved stateHandle to match that of the new response (in error cases, this is true when using actual IDX pipeline). When this is different, it will result in an unexpected introspect request.